model/distro/distro.go

```diff
@@ -45,14 +46,31 @@ func init() {
 
 // GenerateName generates a unique instance name for a distro.
 func (d *Distro) GenerateName() string {
+	// gceMaxNameLength is the maximum length of an instance name permitted by GCE.
+	const gceMaxNameLength = 63
+
 	switch d.Provider {
 	case evergreen.ProviderNameStatic:
```

Comment:
style nit: Don't mix case and if. Since both of these cases return, they can also be if statements.

Reply:
I honestly think that's worse/uglier than what's here. Any reason in particular for this?

Reply:
It just seems odd to mix a case and an if. I'm not married to this though.